### ISSUE TITLE
PHP7 Incompatibility related issues

### ISSUE THREAD

**Issue by soleo** [zikula/core#2737]
With PHP7, mysql extenstion is fully deprecated, so in the installing process, only checking for ``mysql_connect`` is not enough, instead should add checking ``mysqli_connect`` .

```
lib/Zikula/Bundle/CoreInstallerBundle/Form/Type/DbCredsType.php
```

PHP7 also reserved "true", True cannot be the class name. Ref: http://php.net/manual/en/reserved.other-reserved-words.php
This is causing 500 error and the message it returned in router module is that 

```
Compile Error: Cannot use 'True' as class name as it is reserved
```

**Comment by Guite** [comment]
Hi @soleo 
thanks for reporting.

Would you mind testing the fix proposed in #2738 ?

**Comment by soleo** [comment]
It works, I've tested it. I think it's safe to close it now.
